active/ScrollingView/ScrollingView.md

```diff
@@ -0,0 +1,2584 @@
+
+By default, a `ScrollingView` has [ScrollBar](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.Primitives.ScrollBar) controls that allow the user to move the content within the `ScrollingView` using the mouse or pen.
+
+### Using the default Auto settings
+
+The default value of the Horizontal/VerticalScrollBarVisibility properties is 'Auto'. The following pictures show examples of this default behavior for the scroll bars in a `ScrollingView`. 
+
+1.  The `ScrollingView`'s content is arranged to stay within the width of the viewport but not the height. Only the vertical scroll bar appears.
+
+![vertical scroll bar](images/VerticalScrollBar.png)
+
+2.  Because the `ScrollingView`'s content is arranged larger than the viewport in both directions, the two scroll bars and their separator appear.
+
+![both scroll bars and separator](images/BothScrollBars.png)
+
+
+### Hiding both scroll bars
+
+In this example the scroll bars are hidden, meaning that it won't be possible to scroll the content with scroll bars. Scrolling by other means is still permitted.
+
+*`ScrollingView` (new)*
+
+```xml
+<mux:ScrollingView Width="500" Height="400"
+                  ContentOrientation="Both"
+                  HorizontalScrollBarVisibility="Hidden"
+                  VerticalScrollBarVisibility="Hidden">
+    <Image Source="Assets/LargeParisMap.png"/>
+</mux:ScrollingView>
+```
+
+*ScrollViewer (existing)*
+
+```xml
+<ScrollViewer Width="500" Height="400"
+              HorizontalScrollBarVisibility="Hidden"
+              VerticalScrollBarVisibility="Hidden">
+    <Image Source="Assets/LargeParisMap.png"/>
+</ScrollViewer>
+```
+
+
+# Interactivity examples
+
+This section focuses on properties that let you define the user interaction capabilities with the `ScrollingView`.
+
+## Controlling user's ability to zoom
+Enable zooming through user interaction using the `ZoomMode` property (which is Disabled by default). Changes to the ZoomMode property take effect immediately and may affect an on-going user interaction.
+
+*`ScrollingView` (new)*
+
+```xml
+<mux:ScrollingView Width="500" Height="400" ContentOrientation="Both" ZoomMode="Enabled">
+    <Image Source="Assets/LargeParisMap.png"/>
+</mux:ScrollingView>
+```
+
+*ScrollViewer (existing)*
+
+```xml
+<ScrollViewer Width="500" Height="400" HorizontalScrollBarVisibility="Auto" ZoomMode="Enabled">
+    <Image Source="Assets/LargeParisMap.png"/>
+</ScrollViewer>
+```
+
+## Controlling the Minimum and Maximum Zoom Factor
+Use the `MinZoomFactor` and `MaxZoomFactor` properties to control the amount the user may zoom the content. These properties are ignored if the ZoomMode is Disabled.
+
+*`ScrollingView` (new)*
+
+```xml
+<mux:ScrollingView Width="500" Height="400" ContentOrientation="Both"
+                   ZoomMode="Enabled" 
+                   MinZoomFactor="1.0" MaxZoomFactor="2.0">
+    <Image Source="Assets/LargeParisMap.png"/>
+</mux:ScrollingView>
+```
+
+*ScrollViewer (existing)*
```

Comment:
I'll say that having these sprinkled throughout is confusing for a new reader. Maybe there should be a "converting from ScrollViewer to ScrollingView" guide?

Reply:
I think that's good feedback for when this is copied into docs.microsoft.com. Part of the goal in this doc was to help distinguish truly new APIs from those there were simply copied.